recipes/morphMan/LICENSE

```diff
@@ -0,0 +1,1546 @@
+
```

Comment:
The `LICENSE` is part of the GitHub archive so you don't need to add it here.

Reply:
Thanks @beenje, I was not aware, and I have removed the `LICENSE` in the latest commit.